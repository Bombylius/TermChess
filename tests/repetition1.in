g1 f3 b8 c6 f3 g1 c6 b8 g1 f3 b8 c6 f3 g1 c6 b8 e2 e3 e7 e6 g1 f3 b8 c6 f3 g1 c6 b8 g1 f3 b8 c6 f3 g1 c6 b8 g1 f3 b8 c6 f3 g1 c6 b8 g1 f3 b8 c6 f3 g1 c6 b8 
