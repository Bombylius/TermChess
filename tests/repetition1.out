175921860444160
327680
4611686603885117440
43234885634
175921860444160
327680
4611686603885117440
43234885634
17660905521152
269484032
4679521487814656
327680
4611686603885117440
43234889730
4679521487814656
327680
4611686603885117440
43234889730
4679521487814656
327680
4611686603885117440
43234889730
4679521487814656
327680
4611686603885117440
43234889730
rnbqkbnr/pppp1ppp/4p3/8/8/4P3/PPPP1PPP/RNBQKBNR w KQkq - 16 14
4
